src/types.ts

```diff
@@ -17,3 +17,10 @@ export type MessageCall = {
 export enum Rooms {
     All = 'All'
 }
+
+export type CreateChatMessage = {
```

Comment:
if you have common types please move it to `lib/types` for example `common.ts` or `chat.ts`